### ISSUE TITLE
Correct way to handle a new connection using the same PeerConnection

### ISSUE THREAD

**Issue by Kawottex** [microsoft/MixedReality-WebRTC#565]
Hello,
I am currently building an application in which a Peer 1 can connect to a Peer 2, stop the connection and then connect to a Peer 3.

Is there a way to re-use the same Unity.PeerConnection object for both Peer 2 and 3 (not at the same time, but disconnect from Peer 2 and then connect to Peer 3) or do I need to destroy it and re-create it runtime in order to use it with a new peer ?

Thanks,
Kevin.

**Comment by Korangar** [comment]
I am trying currently to use Uninitalize to reuse the Medialine setup, but it seems like a consecutive call to InitalizeAsync never finishes.

Also it seems unclear whether after Uninitalize the connection component is supposed to be "clean", so it can receive or start a new connection.

**Comment by Korangar** [comment]
Added the following change in MediaLine:

```
        internal void UnpairTransceiver()
        {
            DestroySenderIfNeeded();
            Transceiver = null; // <== Add to fix transceiver not being cleaned up?
        }
```

Seems to work.

**Comment by fibann** [comment]
@Kawottex  In 2.0.0 you can disable and re-enable a PeerConnection component to clear the underlying PC and make a new one. Then you can use it to connect again. 

@Korangar reusing a PeerConnection in versions between 1.0.3 and 2.0 is broken as you found out, and we have refactored the corresponding logic. You might get some of the way to there with your fix but I'd recommend updating to 2.0.0.
